rust/crd/src/lib.rs

```diff
@@ -343,6 +415,10 @@ pub struct DriverConfig {
     pub memory: Option<String>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub volume_mounts: Option<Vec<VolumeMount>>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub node_name: Option<String>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub node_selector: Option<std::collections::BTreeMap<String, String>>,
```

Comment:
+1 on `node_selector`, but `node_name` is a bit of a footgun since it bypasses regular scheduling completely. Instead, add a `node_selector` constraint for `kubernetes.io/hostname: foo`

Reply:
Doesn't look like these are actually used anywhere anyway, right?

Reply:
Not yet. This is why this PR is draft. :)